docs/source/receive-adapter.md

```diff
@@ -0,0 +1,156 @@
+# Receive adapter
+
+A _receive adapter_ is a data-plane component which either
+
+- produces [CloudEvents](https://github.com/cloudevents/spec),
+- or imports events and convert them to
+  [CloudEvents](https://github.com/cloudevents/spec),
+
+and forwards those events to one or several consumers (aka _sink_),
+depending upon whether it supports multiple resources (see below for more information).
+
+## Implementating receive adapter
+
+Knative Eventing provides a library (refer later as *the library*) for bootstrapping receive adapter implementations.
+
+The main code should be something like this:
+
+```go
+package main
+
+import (
+  "knative.dev/eventing/pkg/adapter/youradapter"
+  "knative.dev/eventing/pkg/adapter/v2"
+)
+
+func main() {
+  adapter.Main("pingsource",
+    youradapter.NewEnvConfig,
+    youradapter.NewAdapter)
+}
+```
+
+`adapter.Main` is responsible for:
+
+- parsing the command flags and initializing the environment configuration
```

Comment:
same here with the comma's and bullet point list

Reply:
Yep, I'd capitalize bullet list entries and probably add a period at the end of the sentences, i.e.:

- Parsing the command flags and initializing the environment configuration.

Reply:
done